package.json

```
@@ -268,7 +268,7 @@
     "resin-cli-visuals": "^1.8.0",
     "resin-compose-parse": "^2.1.3",
     "resin-doodles": "^0.1.1",
-    "resin-multibuild": "^4.12.1",
+    "resin-multibuild": "4.12.2",
```

```suggestion
    "resin-multibuild": "^4.12.2",
```
In this case it does not make a difference though.